### PR TITLE
Oe 360 required asterisks when adding decision tree node

### PR DESCRIPTION
**Pull request by Zhaolian-Zhou**
As there are different types of decision tree node and with different mandatory fields, didn't show the asterisks at the add page but show a message to remind user enter the required field groups.

### CODE REVIEW

File: protected/modules/OphCoTherapyapplication/views/admin/form_OphCoTherapyapplication_DecisionTreeNode.php [hunk 1]
```diff
@@ -23,19 +23,48 @@
...
-<?php echo $form->dropdownlist($model, 'outcome_id', 'OphCoTherapyapplication_DecisionTreeOutcome', array('empty' => '- Please select -')); ?>
-
-<?php
-$func_list = array();
-foreach ($model->getDefaultFunctions() as $func) {
-    $func_list[$func] = $func;
-}
-echo $form->dropdownlist($model, 'default_function', $func_list, array('empty' => '- Please select -')); ?>
+<div class="alert-box alert">
+<p class="note"> Please fill in one of these filed groups
```
Comment (marshl) [inline protected/modules/OphCoTherapyapplication/views/admin/form_OphCoTherapyapplication_DecisionTreeNode.php]: I think "filed" should be "field"

**Comment by Zhaolian-Zhou** [comment]
I developed issue OE-363 under this branch accidentally.

**Comment by marshl** [comment]
Would it be worth increasing the height of the popup to prevent scrollbars from appearing?

File: protected/modules/OphCoTherapyapplication/views/admin/form_OphCoTherapyapplication_DecisionTreeNode.php [hunk 2]
```diff
@@ -23,19 +23,47 @@
...
-<?php echo $form->dropdownlist($model, 'outcome_id', 'OphCoTherapyapplication_DecisionTreeOutcome', array('empty' => '- Please select -')); ?>
-
-<?php
-$func_list = array();
-foreach ($model->getDefaultFunctions() as $func) {
-    $func_list[$func] = $func;
-}
-echo $form->dropdownlist($model, 'default_function', $func_list, array('empty' => '- Please select -')); ?>
+<div class="alert-box alert">
+<p class="note"> Please fill in one of these field groups
```
[the remark this answers is not in the document]
Comment (Zhaolian-Zhou) [reply protected/modules/OphCoTherapyapplication/views/admin/form_OphCoTherapyapplication_DecisionTreeNode.php]: Fixed

**Review by marshl — APPROVED** [review]

**Pull request merged**